dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -728,6 +728,21 @@
             "modelAsString": true
           }
         },
+        "documentType": {
+          "type": "string",
+          "description": "Document type that can be provided as input for Fhir Documents.",
+          "enum": [
+            "None",
```

Comment:
Do we really need None? Otherwise what is the expectation of using None vs sending empty ?